parser/index.ts

```diff
@@ -38,7 +38,7 @@ function parseHTMLToTokens(html: string): Token[] {
     }
     const close = html.indexOf('>', open);
     if (close === -1) {
-      throw new Error('Invalid HTML: no matching ">"');
+      throw new Error('[react-native-live-markdown] Invalid HTML: no matching ">"');
```

Comment:
Let's add `Error in function parseHTMLToTokens:` for consistency with other error messages.